src/backend/gl/src/info.rs

```diff
@@ -237,19 +238,28 @@ impl Info {
 
 /// Load the information pertaining to the driver and the corresponding device
 /// capabilities.
-pub fn get(gl: &gl::Gl) -> (Info, Features, Limits, PrivateCaps) {
+pub fn query_all(gl: &gl::Gl) -> (Info, Features, Limits, PrivateCaps) {
     use self::Requirement::*;
     let info = Info::get(gl);
     let tessellation_supported =           info.is_supported(&[Core(4,0),
                                                                Ext("GL_ARB_tessellation_shader")]);
     let multi_viewports_supported =        info.is_supported(&[Core(4,1)]); // TODO: extension
-    let max_work_group = get_usize(gl, gl::MAX_COMPUTE_WORK_GROUP_COUNT);
```

Comment:
I mixed up the group count and group size terms 😟 
What I wanted was the maximum number of workgroups launched per dispatch per dimension, which GL and Vk specify as `Count` but I called it `Size` (denoting size of each workgroup).
We need to add the other one as well as it's more important in practical use imo and current checks might need to use the correct value then.